### PR TITLE
Fix: work with nmcli

### PR DESCRIPTION
**Pull request by rusko124**
These edits helps for users w/out network-manager
Refs #1100, https://github.com/Screenly/screenly-ose/issues/1094#issuecomment-488649624

But as I said here https://github.com/Screenly/screenly-ose/issues/1094#issuecomment-488661111
https://github.com/balena-io/wifi-connect uses NetworkManager. In other words, the user will not be able to configure the wireless network.

Sum up: Yes , this PR will help users without NetworkManager, but I think we need to make NetworkManager by default.
@vpetersson Need your opinion.

UPD: Our latest image already includes NetworkManager.

### CODE REVIEW

File: lib/utils.py [hunk 1]
```diff
@@ -120,7 +120,11 @@ def nmcli_get_connections(pattern=None, pattern_ignore=None, fields=None, active
     if active:
         params.append('--active')
 
-    out = sh.nmcli('-t', '-f', ','.join(fields), 'c', 'show', *params)
+    try:
+        out = sh.nmcli('-t', '-f', ','.join(fields), 'c', 'show', *params)
```
Comment (vpetersson) [inline lib/utils.py]: Should we not use dbus for this instead?

File: ansible/roles/network/tasks/main.yml [hunk 2]
```diff
@@ -120,60 +115,33 @@
...
+    dest: /var/lib/polkit-1/localauthority/10-vendor.d/org.freedesktop.NetworkManager.pkla
+
+- name: Set ResultAny to yes
+  replace:
+    regexp: '^ResultAny=.*'
+    replace: 'ResultAny=yes'
+    dest: /var/lib/polkit-1/localauthority/10-vendor.d/org.freedesktop.NetworkManager.pkla
+
+- name: Copy org.freedesktop.NetworkManager.pkla to 50-local.d
+  command: cp -f /var/lib/polkit-1/localauthority/10-vendor.d/org.freedesktop.NetworkManager.pkla /etc/polkit-1/localauthority/50-local.d
```
Comment (vpetersson) [inline ansible/roles/network/tasks/main.yml]: Shoulnd't we use the native copy feature for this?

Comment (rusko124) [reply ansible/roles/network/tasks/main.yml]: It makes sense, but in this case it does not work unfortunately.

Comment (vpetersson) [reply ansible/roles/network/tasks/main.yml]: Got it. All good then. Let's fix the CI run before we can land it tho.

**Pull request merged**